planning/autoware_static_centerline_generator/package.xml

```diff
@@ -33,7 +34,7 @@
   <depend>autoware_vehicle_info_utils</depend>
   <depend>geometry_msgs</depend>
   <depend>global_parameter_loader</depend>
-  <depend>map_loader</depend>
+  <depend>osqp_interface</depend>
```

Comment:
The following error occurred.

```bash
+ rosdep install -y --from-paths src --ignore-src --rosdistro humble
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
autoware_static_centerline_generator: Cannot locate rosdep definition for [osqp_interface]
```

I believe that this line is not necessary.

Reply:
Thank you for pointing out!
I fixed it in https://github.com/autowarefoundation/autoware.universe/pull/8927/commits/6618d30cc70f8ceef1f7de0bdda9a00d40564b5a .